src/client/src/rt-components/resizer/Resizer.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from 'react'
+import { styled } from 'rt-theme'
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+`
+
+const Resizable = styled.div<{ height: number }>`
+  height: ${({ height }) => height + '%'};
+  overflow-y: hidden;
+`
+
+const Bar = styled.div`
+  background: #000;
+  opacity: 0.2;
+  z-index: 1;
+  box-sizing: border-box;
+  background-clip: padding-box;
+
+  height: 11px;
+  margin: -5px 0;
+  border-top: 5px solid rgba(255, 255, 255, 0);
+  border-bottom: 5px solid rgba(255, 255, 255, 0);
+  cursor: row-resize;
+  width: 100%;
+
+  &:hover {
+    border-top: 5px solid rgba(0, 0, 0, 0.5);
+    border-bottom: 5px solid rgba(0, 0, 0, 0.5);
+    transition: all 200ms ease;
```

Comment:
A transition on a box-shadow costs less than on a border because a border affects layout and compositing.